lib/document/RawDocumentInterface.js

```diff
@@ -5,4 +5,5 @@
  * @property {string} $scopeId
  * @property {number} $rev
  * @property {number} $action
+ * @property {RawDocumentMeta|undefined} $meta
```

Comment:
I would keep these names consistent. Rename this one to `RawMetadata` or `Metdata` to `DocumentMetadata`.